### PR TITLE
Enable partially implemented Intel HW intrinsics ISAs - CoreFX part of changes

### PR DESCRIPTION
**Pull request by 4creators**

### CODE REVIEW

**Comment by 4creators** [comment]
CoreCLR part of changes is in PR https://github.com/dotnet/coreclr/pull/17134

**Comment by 4creators** [comment]
Failures are unrelated to this PR: 

On `Windows.10.Amd64.Open-x86` and `Windows.10.Nano.Amd64.Open-x64` failing test is:
```
PreAuthenticate_FirstRequestNoHeader_SecondRequestVariousStatusCodes_ThirdRequestPreauthenticates(statusCode: ResetContent)
```

**Comment by 4creators** [comment]
@CarolEidt @eerhardt @fiigii @tannergooding  

PTAL - PR is ready for review. It is not dependent on any other PR, however, full change will require waiting for changes from CoreCLR to propagate. CI Job failures are unrelated - see comment above.

File: src/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.cs [hunk 1]
```diff
@@ -853,17 +697,6 @@ public enum FloatComparisonMode : byte
...
-        EqualAny = 0x00,
-        Ranges = 0x04,
-        EqualEach = 0x08,
-        EqualOrdered = 0x0c,
-        NegativePolarity = 0x10,
-        MaskedNegativePolarity = 0x30,
-        LeastSignificant = 0x00,
-        MostSignificant = 0x40,
-    }
     public enum ResultsFlag : byte
```
Comment (fiigii) [inline src/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.cs]: `ResultsFlag` also can be removed.

Comment (4creators) [reply src/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.cs]: Thanks. Will fix.

**Comment by eerhardt** [comment]
This change should be against `master`.

We will add these APIs back into `master` once `master` has moved off of `2.1` versioning.  I'm told that the final integration from `master` into `release/2.1` isn't that far off.  So it will only be temporary that these APIs don't exist in master.

**Comment by 4creators** [comment]
Closing as this PR has been replaced with #28443

**Pull request closed without merging**